### PR TITLE
Create a burger menu for links on mobile

### PR DESCRIPTION
**Pull request by vbetivu**
Synopsis
---
Created a burger menu to fix the links overflow on <440px viewport width.

How to test
---
1. Open the homepage.
2. Resize the viewport to <440px width.
3. Observe burger menu.
4. Resize the viewport to >440px width.
5. Observe the previous implementation.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -2,9 +2,7 @@
   <div class="container sidebar-sticky">
     <div class="sidebar-about">
       <h1>
-        <a href="/">
-          {{ site.title }}
-        </a>
+        <a href="/">{{ site.title }}</a>
```
Comment (ujifgc) [inline _includes/header.html]: Please revert the changes not related to the feature.

File: _includes/header.html [hunk 2]
```diff
@@ -19,13 +17,62 @@ <h1>
...
       <ul class="navbar-nav">
         <li class="nav-item"><a class="nav-link" href="/live">live</a></li>
-        <li class="nav-item"><a class="nav-link" href="/repertoire/All">repertoire</a></li>
-        <li class="nav-item"><a class="nav-link" href="/archive/recent">archive</a></li>
+        <li class="nav-item">
+          <a class="nav-link" href="/repertoire/All">repertoire</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="/archive/recent">archive</a>
+        </li>
```
Comment (ujifgc) [inline _includes/header.html]: Same

File: _includes/header.html (hunk 1 above)
Comment (vbetivu) [reply _includes/header.html]: Reverted. The changes relate to the code-formatting, it would be nice to have an explicit formatting configuration, to avoid divergences.
